ios/Instabug.framework/Headers/Instabug.h

```diff
@@ -321,6 +321,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (class, atomic, assign) IBGSDKDebugLogsLevel sdkDebugLogsLevel;
 
+
+#pragma mark - Private Methods for CrossPlatform   /// Will  be removed before the release
```

Comment:
These shouldn't be in the header. Can you please check with iOS about this?

cc: @Kmohamed

Reply:
@HeshamMegid we will remove it in this release